pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -114,6 +116,13 @@ func ConvertCgroupFsNameToSystemd(cgroupfsName string) (string, error) {
 	return path.Base(cgroupfsName), nil
 }
 
+func IsSystemdStyleName(name string) bool {
```

Comment:
nit: should we just have a /kubelet/cm/systemd subpackage in future and move this there?  it will be odd when _windows is having to implement this.